esb-connector-creatio-crm/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright 2018 WSO2, Inc. (http://wso2.com)
```

Comment:
Let's change the license header (the year)

Reply:
Updated.